org.kde.okular.json

```diff
@@ -287,6 +287,23 @@
             }
          ]
       },
+      {
+         "name": "okular-wrapper",
+         "buildsystem": "simple",
+         "build-commands": [
+            "install -Dm755 okular-wrapper /app/bin/okular-wrapper"
+            ],
+         "sources": [
+            {
+                "type": "script",
+                    "dest-filename": "okular-wrapper",
+                    "commands": [
+                        "export TMPDIR=\"$XDG_RUNTIME_DIR/app/$FLATPAK_ID\"",
```

Comment:
TMPDIR=$XDG_CACHE_HOME

Reply:
`$XDG_CACHE_HOME` isn't tmpfs and may accumulate cruft. I intentionally used different path